src/Sql/Sql/ThreatDetection/Model/DetectionType.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Sql.ThreatDetection.Model
+{
+    /// <summary>
+    /// The possible disable alert types
+    /// </summary> 
+    public static class DetectionType
```

Comment:
Are these values available in .net sdk (generated from swagger)? There should not be a need to define them in powershell layer.

Reply:
I just checked and I don't see them in Swagger. This is a good change for powershell code, but you really should update Swagger by defining enum so that the allowed values are defined as constants in sdk layer instead of needing to define it here in powershell layer.

Reply:
if we define it as enum in swagger we won't be able to pass a free string to the .Net client, isn't that true ?

Reply:
Not true. Enums in swagger can be generated (in C#, for example) as either enums or as const strings. This is controlled by `modelAsString` property in swagger.